projects/plugins/jetpack/extensions/blocks/google-calendar/controls.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * WordPress dependencies
+ */
+import { Button, PanelBody } from '@wordpress/components';
+import { __, _x } from '@wordpress/i18n';
+
+export const GoogleCalendarInspectorControls = ( props ) => {
+	const { className, embedValue, onChange, onSubmit } = props;
+
+	return (
+		<PanelBody title={ __( 'Calendar settings', 'jetpack' ) } initialOpen={ false }>
+			<form onSubmit={ onSubmit } className={ className }>
+				<textarea
+					type="text"
+					value={ embedValue }
+					className="components-placeholder__input"
+					aria-label={ __( 'Google Calendar URL or iframe', 'jetpack' ) }
+					placeholder={ __( 'Enter URL or iframe to embed here…', 'jetpack' ) }
+					onChange={ onChange }
+				/>
+				<Button isSecondary isLarge type="submit">
```

Comment:
The `isLarge` attribute was already present before the refactor, but I can see it's throwing a warning when running in dev mode. Shall we remove it here?

![image](https://user-images.githubusercontent.com/14988353/113828958-4f065300-97c8-11eb-8437-999416258cfe.png)

Reply:
Thanks. The occurrences of `isLarge` have been removed.